modules/boost.program_options/1.83.0/overlay/test/MODULE.bazel

```diff
@@ -0,0 +1,7 @@
+bazel_dep(name = "boost.program_options")
+local_path_override(
```

Comment:
This will be ignored, are you sure you want to check it in?

Reply:
You mean `local_path_override` will be skipped, right?

Reply:
Yes

Reply:
I'm not sure why it is ignored, is there any way to fix it?

Reply:
Overrides can only be applied by the root module. What do you want to achieve with this override?

Reply:
https://github.com/wep21/bazel-central-registry/blob/20feeda51c290d8d164954da8daf0055549bed43/modules/boost.program_options/1.83.0/presubmit.yml#L21
In the test phase, module path is pointed to `test` directory, so I've thought `test/MODULE.bazel` becomes the root module in the test phase. Is that wrong?

Reply:
This should actually work. Thanks for the explanation.